### ISSUE TITLE
Show needinfo turnaround time

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#2184]
We could show the evolution of # of needinfos, and the evolution of median needinfo turn around times.

**Comment by jrbenny35** [comment]
Does this need to be added to the json blob?

**Comment by marco-c** [comment]
@jrbenny35 yes, it's not feasible without touching the script that generates the blob too.
